hack/test-cmd.sh

```diff
@@ -78,9 +84,15 @@ set -e
 
 function find_tests {
   find "${OS_ROOT}/test/cmd" -name '*.sh' -not -wholename '*images_tests.sh' | grep -E "${1}" | sort -u
```

Comment:
Ah - I see what you mean. `cleanup` (and `cleanup_openshift`) should both run with `set +e` for now. This is not particularly great, but since they're best-effort cleanups we can let it slide until a refactor. A correct implementation would, as you say, check before jumping and not fail if things are unset. That being said, that's out of scope for this PR. Having a nice error message come out from this logic that happens to trigger cleanup as well is OK.